frontend/probloom/src/components/Prob/Prob.tsx

```diff
@@ -0,0 +1,28 @@
+export interface ProblemProps {
+    title: string;
+    date: string;
+    creator: string;
+    solved: number;
+    recommended: number;
+    clickProb: () => void;
+}
+
+const Prob = (props: ProblemProps) => {
```

Comment:
If you named this component `Prob` because `Problem` is already used in `problemReducer`, I think it might be better to use something like `ProblemSearchResult` instead of `Prob`. In my opinion, it is hard to know what `Prob` is just by seeing the name of it.